apps/client/src/app/core/data/localized-lazy-data.service.ts

```diff
@@ -0,0 +1,463 @@
+import { Injectable } from '@angular/core';
+import { get } from 'lodash';
+import { Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { I18nLazy, I18nNameLazy } from '../../model/common/i18n-name-lazy';
+import { Fate } from '../../pages/db/model/fate/fate';
+import { ExtendedLanguageKeys } from './extended-language-keys';
+import { Language } from './language';
+import { LazyDataKey } from './lazy-data';
+import { LazyDataProviderService } from './lazy-data-provider.service';
+import { lazyFilesList } from './lazy-files-list';
+import { mapIds } from './sources/map-ids';
+import { zhWorlds } from './sources/zh-worlds';
+
+@Injectable({ providedIn: 'root' })
+export class LocalizedLazyDataService {
```

Comment:
Maybe this should implement LazyDataService, to make sure they expose the same methods over time.

Reply:
Hm, I wonder if that might be a little awkward, since the methods have different signatures? Do you think they should all be suffixed with something, maybe (like `getItemLazy(id: number): Observable<I18nNameLazy>`)?

Reply:
Hm indeed, This means that either we create an interface with a generic type for returns but it'll be a pain, or we let them as they are right now.

Yeah the latter is the better I think here 🙂